components/compositing/compositor.rs

```diff
@@ -1940,7 +1942,9 @@ impl<Window: WindowMethods> IOCompositor<Window> {
 
     fn find_topmost_layer_at_point_for_layer(&self,
                                              layer: Rc<Layer<CompositorData>>,
+                                             // The point in the parent layer's coordinates
                                              point: TypedPoint2D<LayerPixel, f32>,
```

Comment:
I wonder if it would make sense to simply call this point_in_parent_layer and remove the comment.